native_client/client.cc

```diff
@@ -34,13 +34,17 @@ typedef struct {
 
 ds_result
 LocalDsSTT(ModelState* aCtx, const short* aBuffer, size_t aBufferSize,
-           int aSampleRate)
+           int aSampleRate, bool extended_output)
 {
   ds_result res = {0};
 
   clock_t ds_start_time = clock();
 
-  res.string = DS_SpeechToText(aCtx, aBuffer, aBufferSize, aSampleRate);
+  if (extended_output) {
+	  res.string = DS_SpeechToTextExtended(aCtx, aBuffer, aBufferSize, aSampleRate);
```

Comment:
This no longer builds with the API changes since DS_SpeechToTextExtended returns a Metadata struct rather than a char*. Are you planning to do the client changes in a separate PR or will you add commits here?